tmeister/static/webpack.config.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-env node */
+const webpack = require('webpack')
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  entry: path.resolve(__dirname, 'src/toggle-meister.js'),
+  output: {
+    filename: 'toggle-meiser.js',
+    path: path.resolve(__dirname, 'build'),
+  },
+  mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.js?$/,
+        exclude: [path.resolve(__dirname, 'node_modules')],
+        loader: 'babel-loader',
+        options: {
+          plugins: [
+            // // https://github.com/babel/babel-loader#babel-is-injecting-helpers-into-each-file-and-bloating-my-code
+            // "@babel/plugin-transform-runtime",
```

Comment:
confused here... do we want `@babel/plugin-transform-runtime` to be active or not? If not, it seems the package should be removed from `package.json`. Otherwise, it seems that line 23 should not be commented.

Reply:
I think I commented it out and forgot to put it back.

Reply:
actually, this should probably just be in included/added in the .babelrc file